lib/rspec_profiling/collectors/statsd.rb

```diff
@@ -0,0 +1,100 @@
+require "statsd";
+require "logger";
+require "open3";
+
+
+
+module RspecProfiling
+  module Collectors
+    class Statsd
+      #delegate :timing, :increment, :count, :gauge, :time, :batch, to: :statsd, allow_nil: true 
+      NAMESPACE = 'rspec_profiling'
+      #Property used by CSV Collector, leaving it here
+      #to maintain reference of available properties.
+      HEADERS = %w{
+        branch
+        commit_hash
+        date
+        file
+        line_number
+        description
+        status
+        exception
+        time
+        query_count
+        query_time
+        request_count
+        request_time
+      }
+
+      attr_accessor :statsd, :results, :logger
+      
+      def self.install
+        new.install
+      end
+
+      def self.uninstall
+        new.uninstall
+      end
+
+      def health(host, port)
+        `nc -vuzn #{host} #{port} 2>&1 > /dev/null`.include? 'open'
```

Comment:
code style: all method calls should have parentheses, so should be `.include?('open')`